<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sensor_act">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>s1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="schuifdeurKnop">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>vd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draaideurKnop1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>d1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draaideurKnop2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>d2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQing">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPiep">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2023">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPiep">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Code: Piep</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2023">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Code: 2023</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelQing">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Code: Qing</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOOPR">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelOOPR">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Access kaart OOPR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2020">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label2020">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Access kaart 2020</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAddPerson">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdd">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Person</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allowKnop">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="denyKnop">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Deny</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="kaartenbak">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="kaartenbakKnop">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Kaartenbak</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
